app/components/channel_intro/channel_intro.js

```diff
@@ -69,7 +69,7 @@ class ChannelIntro extends PureComponent {
         return currentChannelMembers.map((member) => (
             <TouchableOpacity
                 key={member.id}
-                onPress={() => preventDoubleTap(this.goToUserProfile, this, member.id)}
+                onPress={preventDoubleTap(() => this.goToUserProfile(member.id))}
```

Comment:
In the future, I think we should extract those `buildSomething` into its won separate component.